rampwf/externals/tabulate.py

```diff
@@ -2,18 +2,21 @@
 
 """Pretty-print tabular data."""
```

Comment:
I think this is from https://github.com/astanin/python-tabulate we shouldn't really migrate it ourselves, it would be better to just include a more recent version from that repo. If they didn't change the API too much.

Reply:
Indeed, I have include the latest formal version 0.8.10 instead of modifying the code myself.